documentation/modules/ref-address-example-forwarders.adoc

```diff
@@ -6,19 +6,20 @@
 = Address forwarding examples
 
 You can use forwarders to:
-* automatically forward messages from a "local" address to a remote AMQP address outside of {ProductName}, or 
-* forward messages from a remote AMQP address to a "local" address. 
-To create a forwarder you must first configure a connector to the remote AMQP endpoint for the address space. For more information about connectors, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-address-space-example-connectors-messaging[Address space connector examples].
 
-Forwarding is only supported in the `standard` address space type, and only for the `queue` and `subscription` address types. The `subscription` address type only supports forwarding in the `out` direction.
+* automatically forward messages from a local address to a remote AMQP server outside of {ProductName}, or
 
-These examples expect that a connector `remote1` has been configured for the address space.
+* forward messages from a remote AMQP server to a local address.
 
-== Forwarding messages from a local queue to a remote endpoint
+To use an address forwarder, you must first configure a connector to the remote AMQP server for the address space. For more information about address space connectors, see link:{BookUrlBase}{BaseProductVersion}{BookNameUrl}#ref-address-space-example-connectors-messaging[Address space connector examples].
 
-In this example, messages in `myqueue` are forwarded to the remote endpoint with an  address of `clients/me/1`.
+Address forwarding is only supported in the `standard` address space type, and only for the `queue` and `subscription` address types. With the `queue` address type, messages can be forwarded to a remote AMQP server or messages can be forwarded from a remote AMQP server to a local queue. With the `subscription` address type, you can create a forwarder to a remote AMQP address, but you cannot create a forwarder that copies messages to the subscription. That is, the `subscription` address type only supports forwarding in the `out` direction, as shown in the example.
```

Comment:
"....the subscription` address type supports forwarding in only the `out` direction.."